### PR TITLE
Documentation for the statistical model

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #1825

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 32 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 51 sec

File: docs/src/administrator/manual/django-change-data.md [hunk 1]
```diff
@@ -132,3 +132,27 @@ The other options (`History`, `Delete`, `Save and add another`, `Save and contin
...
+
+![Change Statistical Model](./img/django-change-data-13.png)
+
+1. **History**: Button to view actions applied to the current record.
+
+2. **Form Fields**: Statisticians have the ability to update the values of various fields within the form. However, it's important to note that the "Taxon" field is not editable. If a statistician wishes to change the taxon associated with a statistical model, they need to create a new statistical model specifically for the desired taxon. This restriction ensures data integrity and alignment with the taxonomic hierarchy, requiring a distinct model for each taxon to accurately capture and represent statistical nuances unique to that taxon.
+
+![Change Statistical Model](./img/django-change-data-12.png)
+
+3. **Delete Checkbox**: Check mark the checkbox to delete the associat statistical output type.
```
Comment (Jeremy-Prior) [inline docs/src/administrator/manual/django-change-data.md:146]:
```suggestion
3. **Delete Checkbox**: Check mark the checkbox to delete the associated statistical output type.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 17 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 19 sec

**Pull request merged**